CHANGELOG.rst

```diff
@@ -13,13 +13,15 @@ Added
 -----
 - ``TrainingFileImporter`` interface to support customizing the process of loading
   training data
-- Fill slots for custom templates
+- fill slots for custom templates
+- new event broker class: ``SQLProducer``. This event broker is now used when running locally with
+  Rasa X
```

Comment:
just a reminder to make sure this gets merged in the right place when you merge in latest master (1.1.8 was already release)